src/common/talents/TALENTS_PRIEST.js

```diff
@@ -17,7 +17,7 @@ export default {
   SHADOW_COVENANT_TALENT: { id: 204065, name: "Shadow Covenant", icon: "spell_shadow_summonvoidwalker", baseMana: 0.05 },
   DIVINITY_TALENT: { id: 197031, name: "Divinity", icon: "ability_priest_ascendance" },
   DIVINE_STAR_TALENT: { id: 110744, name: "Divine Star", icon: "spell_priest_divinestar", baseMana: 0.02 },
-  HALO_TALENT: { id: 120517, name: "Halo", icon: "ability_priest_halo" },
+  HALO_TALENT: { id: 120517, name: "Halo", icon: "ability_priest_halo", baseMana: 0.027 },
```

Comment:
This will be automatically overwritten when I generate the file next; if you want to override properties from the auto generated files you can redefine the spells in the `src/common/SPELLS_PRIEST.js` file. I'll work on making the talent generated work with partial mana cost numbers.